dmoj/graders/bridged.py

```diff
@@ -3,6 +3,7 @@
 import subprocess
 
 from dmoj.contrib import contrib_modules
+from dmoj.error import CompileError
```

Comment:
Can we merge this with the line below (so `from dmoj.error import CompileError, InternalError`